x-pack/plugins/metrics_entities/server/scripts/post_examples/all.json

```diff
@@ -0,0 +1,32 @@
+{
+  "modules": [
+    "host_metrics",
+    "host_entities",
+    "network_metrics",
+    "network_entities",
+    "user_entities",
+    "user_metrics"
+  ],
+  "indices": [
+    "auditbeat-*",
+    "endgame-*",
+    "filebeat-*",
+    "logs-*",
+    "packetbeat-*",
+    "winlogbeat-*",
+    "-logs-diagnotic*",
```

Comment:
Just as is removed in the UI from Sourcerer -- 👍

Reply:
Yeah, that was an old index oddness. I deleted that on our test servers and it should not be here anymore. Will remove it from these examples.